inputs/select.vue

```diff
@@ -35,7 +35,8 @@
 
   > #### info::Usage Notes
   >
-  > * no/empty option is pre-selected by default (you don't need to specify an empty option in the schema)
+  > * you can have a value for a 'None' option, this is useful for components that have defaults that you want to be able to revert to after selecting an option
```

Comment:
I would add an example showing this in action, since it's a little confusing at first glance

Reply:
@nelsonpecora Is this what you meant? https://github.com/clay/clay-kiln/pull/1161/files#diff-15e2547214bfb6b1eba657f0bb1b5c91R50

Reply:
yes!